code/game/gamemodes/blob/blobs/core.dm

```diff
@@ -12,19 +12,19 @@
 	var/is_offspring = null
 	var/selecting = 0
 
-/obj/structure/blob/core/New(loc, var/h = 200, var/client/new_overmind = null, var/new_rate = 2, offspring)
```

Comment:
This `h` var was never used anywhere, so I fail to see why it was passed all the way down